src/server/routes.rs

```diff
@@ -26,6 +26,8 @@ pub fn config<T: Store + 'static>(cfg: &mut ServiceConfig) {
                 resource("/login")
                     .route(get().to(handlers::auth::login_info))
                     .route(post().to(handlers::auth::login::<T>)),
-            ),
+            )
+            .service(resource("/account").route(get().to(handlers::account::whoami)))
+            .wrap(AuthChecker::new()),
```

Comment:
I'm curious: Does this affect to all routes? It's possible to register some routes with this middleware and some other routes without?

Reply:
Yes it is possible to do this on a resource level. That actually make sense here as some of the methods require different auth